_docs/reference/config/mixer/attribute-vocabulary.md

```diff
@@ -63,4 +63,8 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | connection.sent.bytes_total | int64 | Total number of bytes sent by a destination service during the lifetime of a connection. | |
 | connection.duration | duration | The total amount of time a connection has been open. | |
 | context.protocol | string | Protocol of the request or connection being proxied. | tcp |
-| context.time | timestamp | The timestamp of the Report() call. | |
\ No newline at end of file
+| context.time | timestamp | The timestamp of the Report() call. | |
+| api.service | string | The public service name. One API could be handled by multiple internal services. Or multiple APIs could be handled by a single internal service. | my-svc.com |
+| api.version | string | The API version. Not required. | v1alpha1 |
```

Comment:
i don't think we need "Not required." here. This isn't about how attributes are used or when, but just about their names and definitions.

Reply:
Done.